framework/views/errorHandler/exception.php

```diff
@@ -345,44 +347,50 @@
             <h1><?php
-                if ($exception instanceof \yii\web\HttpException) {
```

Comment:
This is what `PHPCBF` thinks it should look like. Haven't reviewed this myself yet.